src/maud/inference_model.stan

```diff
@@ -34,8 +34,8 @@ data {
   int<lower=1,upper=N_reaction> reaction_yflux[N_flux_measurement];
   real yflux[N_flux_measurement];
   vector<lower=0>[N_flux_measurement] sigma_flux;
-  int<lower=1,upper=N_experiment> experiment_yenz[N_enzyme_measurement];
-  int<lower=1,upper=N_enzyme> enzyme_yenz[N_enzyme_measurement];
+  int<lower=0,upper=N_experiment> experiment_yenz[N_enzyme_measurement];
+  int<lower=0,upper=N_enzyme> enzyme_yenz[N_enzyme_measurement];
```

Comment:
Is this right? Stan uses 1-indexing so if these are 0-indexed then something will probably break

Reply:
Yes, because it's possible to have no measurements if we define everything as priors for example